cluster/gce/gci/master.yaml

```diff
@@ -74,6 +74,33 @@ write_files:
       [Install]
       WantedBy=kubernetes.target
 
+  - path: /etc/systemd/system/kube-logrotate.timer
+    permissions: 0644
+    owner: root
+    content: |
+      [Unit]
+      Description=Hourly kube-logrotate invocation
+
+      [Timer]
+      OnCalendar=hourly
+
+      [Install]
+      WantedBy=kubernetes.target
+
+  - path: /etc/systemd/system/kube-logrotate.service
```

Comment:
Since this is an one-shot job and it depends on configure-helper.sh, can we move the line "/usr/sbin/logrotate /etc/logrotate.conf" into the configure-helper.sh and get rid of this systemd job?

Two benefits: (1) Less jobs here, easier for people to understand. This is the feedback I got from some k8s developers; (2) The logic is the same for both master and node, we can use just one line in configure-helper.sh to do it, instead of two write-file sections.